src/style.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Poppins:wght@400;500;600&display=swap');
+
+* {
+  padding: 0;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+}
+
+h1 {
+  text-align: center;
+}
+
+ul.list__placeholder {
+  list-style-type: none;
+}
+
+ul.list__placeholder li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 10px;
+  border: 1px solid rgb(207, 201, 201);
+  margin-bottom: 20px;
+}
+
+ul.list__placeholder li input {
+  margin-right: 1.2rem;
+  cursor: pointer;
+}
+
+ul.list__placeholder li i {
+  cursor: pointer;
+}
```

Comment:
- Hey, great job with the styling, however, you're design is not looking close to the design from the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Please update your styles to look like the design in the video above. See the images below to compare your design:



*design on video*
![Screenshot from 2023-01-26 15-41-35](https://user-images.githubusercontent.com/59397159/214866188-f18f71ea-d4dd-437b-978a-d620461d0d53.png)

*your design*
![Screenshot from 2023-01-26 15-40-28](https://user-images.githubusercontent.com/59397159/214866194-8bdf2748-d380-43db-bc32-366d84fa1415.png)